css/main.css

```diff
@@ -0,0 +1,75 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 700;
+}
+
+::-webkit-scrollbar {
+  width: 6px;
+  height: 10px;
+  display: none;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.logo > img {
+  width: 80px;
+  height: 80px;
+  padding: 10px;
+}
+
+header > .button {
+  padding: 30px;
+}
+
+.hero-section {
+  background-image: url("../image/heroImage.jpg");
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.title {
+  font-size: 30px;
+  margin: auto;
+  text-align: left;
+}
+
+.title > h1 {
+  margin-bottom: 0;
+}
+
+.description {
+  margin: auto;
+  font-size: 14px;
+  text-align: left;
+  font-weight: normal;
+}
```

Comment:
- [ ] Kyndly use the font size, font family and color stablished on the figma design. 

## Figma

![image](https://user-images.githubusercontent.com/95243996/191735298-81fc57eb-5069-4550-8637-3db0619fc780.png)

## Your design 

![image](https://user-images.githubusercontent.com/95243996/191735925-218a7c85-5a71-471c-8e2c-df26f841eaa5.png)